pom.xml

```
@@ -72,6 +76,7 @@
 					<consoleOutput>true</consoleOutput>
 					<failsOnError>true</failsOnError>
 					<linkXRef>false</linkXRef>
+					<sourceDirectories>src</sourceDirectories>
```

wrong way of using sourceDirectories attribute. 
add separately 
src/main/java and src/test/java folders

```suggestion
					<sourceDirectories><sourceDirectory>src/main/java</sourceDirectory></sourceDirectories>
```